pkgs/build-support/bintools-wrapper/default.nix

```
@@ -80,7 +80,7 @@ let
     else if targetPlatform.libc == "nblibc"           then "${sharedLibraryLoader}/libexec/ld.elf_so"
     else if targetPlatform.system == "i686-linux"     then "${sharedLibraryLoader}/lib/ld-linux.so.2"
     else if targetPlatform.system == "x86_64-linux"   then "${sharedLibraryLoader}/lib/ld-linux-x86-64.so.2"
-    else if targetPlatform.system == "powerpc64le-linux" then "${sharedLibraryLoader}/lib/ld64.so.2"
+    else if targetPlatform.isPower64                  then "${sharedLibraryLoader}/lib/ld64.so.2"
```

`ld64.so.2` is the linker for ELFv2 ABI only, ELFv1 uses `ld64.so.1`. So if anything, this would seem more correct to me:

```suggestion
    else if (targetPlatform.isPower64 && targetPlatform.isAbiElfv2) then "${sharedLibraryLoader}/lib/ld64.so.2"
```

But the `isAbiElfv2` inspection doesn't match our default `powerpc64le-linux`'s config because we don't explicitly dictate `elfv2` there (we default to it on big-endian). So that would break powerpc64le.

I think using a wildcard + explanation would work here, seems to behave fine in my (slow) testing:
```suggestion
    # ELFv1 (.1) or ELFv2 (.2) ABI
    else if targetPlatform.isPower64                  then "${sharedLibraryLoader}/lib/ld64.so.*"
```

---

Great catch, thanks a lot! Should be fixed now.